### DISCUSSION TITLE
Help Me!!!! Compiling Problems

### DISCUSSION THREAD

**Discussion by CasualCypress** [FunkinCrew/Funkin#2112] [Compiling help]
## I got all this stuff for doing 'lime test windows -debug'.

hxcpp/debug/jsonrpc/Server.hx:86: Debug Server Started:
hxcpp/debug/jsonrpc/Server.hx:592: Failed to connect to vsc debugger server at 127.0.0.1:6972
Called from hxcpp::__hxcpp_main
Called from ApplicationMain::main ApplicationMain.hx line 26
Called from ApplicationMain::create ApplicationMain.hx line 135
Called from lime.app.Application::exec lime/app/Application.hx line 150
Called from lime._internal.backend.native.NativeApplication::exec lime/_internal/backend/native/NativeApplication.hx line 146
Called from lime._internal.backend.native.NativeApplication::handleApplicationEvent lime/_internal/backend/native/NativeApplication.hx line 175
Called from lime.app._Event_Int_Void::dispatch lime/_internal/macros/EventMacro.hx line 91
Called from lime.system.ThreadPool::__update lime/system/ThreadPool.hx line 199
Called from lime.app._Event_Dynamic_Void::dispatch lime/_internal/macros/EventMacro.hx line 91
Called from lime._internal.backend.native.NativeHTTPRequest::localThreadPool_onComplete lime/_internal/backend/native/NativeHTTPRequest.hx line 450
Called from lime.app.Promise_haxe_io_Bytes::complete lime/app/Promise.hx line 108
Called from lime.net._HTTPRequest_Bytes::load lime/net/HTTPRequest.hx line 114
Called from lime.app.Promise::complete lime/app/Promise.hx line 108
Called from lime.app.Future::then lime/app/Future.hx line 282
Called from lime.app.Future::onComplete lime/app/Future.hx line 127
Called from lime.app.Promise::complete lime/app/Promise.hx line 108
Called from lime.utils.Assets::loadLibrary lime/utils/Assets.hx line 476
Called from lime.app.Promise_lime_utils_AssetLibrary::completeWith lime/app/Promise.hx line 126
Called from lime.app.Future::onComplete lime/app/Future.hx line 127
Called from lime.app.Promise_lime_utils_AssetLibrary::complete lime/app/Promise.hx line 108
Called from lime.utils.Preloader::updateProgress lime/utils/Preloader.hx line 289
Called from lime.utils.Preloader::loadedAssetLibrary lime/utils/Preloader.hx line 204
Called from lime.utils.Preloader::updateProgress lime/utils/Preloader.hx line 309
Called from lime.utils.Preloader::start lime/utils/Preloader.hx line 220
Called from lime.app._Event_Void_Void::dispatch lime/_internal/macros/EventMacro.hx line 91
Called from ApplicationMain::create ApplicationMain.hx line 116
Called from openfl.display.Preloader::start openfl/display/Preloader.hx line 61
Called from lime.app._Event_Void_Void::dispatch lime/_internal/macros/EventMacro.hx line 91
Called from ApplicationMain::create ApplicationMain.hx line 119
Called from ApplicationMain::start ApplicationMain.hx line 169
Called from Main::main Main.hx line 36
Called from openfl.display.DisplayObjectContainer::addChild openfl/display/DisplayObjectContainer.hx line 155
Called from openfl.display.DisplayObjectContainer::addChildAt openfl/display/DisplayObjectContainer.hx line 267
Called from openfl.display.DisplayObject::__dispatchWithCapture openfl/display/DisplayObject.hx line 1478
Called from openfl.display.DisplayObject::__dispatchEvent openfl/display/DisplayObject.hx line 1417
Called from openfl.events.EventDispatcher::__dispatchEvent openfl/events/EventDispatcher.hx line 402
Called from Main::init Main.hx line 60
Called from Main::setupGame Main.hx line 87
Called from openfl.display.DisplayObjectContainer::addChild openfl/display/DisplayObjectContainer.hx line 155
Called from openfl.display.DisplayObjectContainer::addChildAt openfl/display/DisplayObjectContainer.hx line 267
Called from openfl.display.DisplayObject::__dispatchWithCapture openfl/display/DisplayObject.hx line 1478
Called from openfl.display.DisplayObject::__dispatchEvent openfl/display/DisplayObject.hx line 1417
Called from openfl.events.EventDispatcher::__dispatchEvent openfl/events/EventDispatcher.hx line 402
Called from flixel.FlxGame::create flixel/FlxGame.hx line 360
Called from flixel.FlxGame::switchState flixel/FlxGame.hx line 637
Called from TitleState::create TitleState.hx line 73
Called from polymod.Polymod::init polymod/Polymod.hx line 252
Called from polymod.backends.PolymodAssets::init polymod/backends/PolymodAssets.hx line 133
Called from polymod.backends.LimeBackend::init polymod/backends/LimeBackend.hx line 127
Critical Error: Uncatchable Throw: Null Object Reference

Help Me

**Comment by biomseed** [comment]
#2109
